Tests/Integration/CloudToDeviceMessageSizeLimitShouldAbandonTests.cs

```diff
@@ -36,8 +36,10 @@ public class CloudToDeviceMessageSizeLimitShouldAbandonTests : CloudToDeviceMess
             frmCntDown: InitialDeviceFcntDown);
 
             var loraDevice = CreateLoRaDevice(simulatedDevice);
+            var msgPayload = "1234567890";
+            var confirmedMessagePayload = simulatedDevice.CreateConfirmedDataUpMessage(msgPayload, isHexPayload: true, fport: 0);
```

Comment:
CodeQL is issueing a warning for useless assignment here

Reply:
Right this was superseeded by CreateUpstreamMessage local method